es-ds-docs/assets/scss/main.scss

```diff
@@ -0,0 +1,2 @@
+// EnergySage design system styles
+@use '@energysage/es-bs-base/scss/bootstrap.import' as *;
```

Comment:
This can be referenced directly in the nuxt-config without needing a `main.scss`

Reply:
Makes sense! Done.